helpers/setup.sh

```diff
@@ -185,6 +190,18 @@ if [[ -n "$WITH_ENFORCER" ]]; then
   done
 fi
 
+if [[ -n "$ON_GKE" ]]; then
+  gke_roles=("roles/container.admin" "roles/iam.serviceAccountAdmin" "roles/compute.networkAdmin" "roles/resourcemanager.projectIamAdmin")
```

Comment:
The "roles/iam.serviceAccountAdmin" role looks like it is already added in line 147- does it need to be removed here?

Reply:
We can take it out!